adafruit_ina260.py

```diff
@@ -195,27 +207,31 @@ def __init__(self, i2c_bus, address=0x40):
 
     mask_enable = RWBits(16, _REG_MASK_ENABLE, 0, 2, False)
     alert_limit = RWBits(16, _REG_ALERT_LIMIT, 0, 2, False)
+    
+    manufacturer_id = ROUnaryStruct(_REG_MFG_UID, ">H")
```

Comment:
I don't think these three are particularly useful for the user. If you'd like to keep them, please add them as a check in `__init__` that verifies we're talking to the correct chip. 

You can see an example here:
 https://github.com/adafruit/Adafruit_CircuitPython_LSM6DS/blob/master/adafruit_lsm6ds/__init__.py#L206
 
With that in mind, I don't thing we need the manufacturer id or die id. Please remove them and rename `device_id` to `_device_id` to make it clear it's not meant for public consumption.

Reply:
interesting point, I'm going to add this check